en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-1100-to-400.md

```diff
@@ -1459,6 +1459,9 @@ Follow the instructions below to move all the existing API Manager configuration
             ADD BILLING_CYCLE VARCHAR(15) DEFAULT NULL 
             ADD PRICE_PER_REQUEST VARCHAR(15) DEFAULT NULL 
             ADD CURRENCY VARCHAR(15) DEFAULT NULL
+            ADD MAX_COMPLEXITY INT NOT NULL DEFAULT 0
```

Comment:
Can we make all the ADD statements to the same indentation level (comparing line 1458 and 1459)

Reply:
Fixed in c7e069a48c80f05857f478107339b5b153c983a6